### PR TITLE
[v1] Fix code smells

### PR DESCRIPTION
**Pull request by t1m0thyj**
## Proposed changes

Fix code smells identified by Coverity scan

## Release Notes

I don't think this needs to be mentioned in the changelog

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [x] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [ ] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [ ] PR Description is included
- [ ] gif or screenshot is included if visual changes are made
- [x] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

## Further comments

Backports #1937 and #1995 to the v1-lts branch

### CODE REVIEW

**Review by JillieBeanSim — DISMISSED** [review]
LGTM! Thanks @t1m0thyj for the fix

**Review by zFernand0 — APPROVED** [review]
LGTM! 😋

File: .github/workflows/zowe-explorer-ftp-ci.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
       # Continue to run tests on the other systems if one fails
       fail-fast: false
       matrix:
-        node-version: [12.x, 14.x]
+        node-version: [14.x, 16.x]
```
Comment (zFernand0) [inline .github/workflows/zowe-explorer-ftp-ci.yml]: Thanks for updating the workflows 🙏 
We can create a separate issue for addressing any node16 specific problems in the future 😋 
For now I do believe we can continue building with node14 for the v1-lts branch 😀

**Review by zFernand0 — APPROVED** [review]
LGTM! 😋

**Review by JillieBeanSim — APPROVED** [review]
Thanks @t1m0thyj for the fix

**Pull request merged**